### PR TITLE
(tiny edit) now cards can accept uppercase letters as a param too

### PR DESCRIPTION
**Pull request by wmginsberg**
I was trying your cards (which are super cool btw!) and the images on my face cards weren't showing up. I realized they only accepted lowercase chars as parameters. I added a native toLowerCase function, so the component can accept both upper and lowercase letters.

### CODE REVIEW

**Comment by vpusher** [comment]
Nice 👍  ! Thank you.

File: game-card.html [hunk 1]
```diff
@@ -348,7 +348,7 @@
             }
 
             computeFigureImage() {
-                let rank = GameCard.FIGURES.includes(this.rank) && GameCard.RANKS[this.rank],
+                let rank = GameCard.FIGURES.includes(this.rank.toLowerCase()) && GameCard.RANKS[this.rank.toLowerCase()],
```
Comment (vpusher) [inline game-card.html:351]: You probably want to do the same thing in `#computePositionClass` when testing `this.rank === 'a'`.

**Review by vpusher — COMMENTED** [review]
Once this one is fixed, will bump version and merge.

File: game-card.html (hunk 1 above)
Comment (wmginsberg) [reply game-card.html:351]: ahh yeah, good catch :) added

**Pull request merged**